com.civicrm.hrjobroles/views/include/add_new_role.html

```diff
@@ -169,169 +169,177 @@
-          <button type="button" class="btn btn-link pull-right" ng-show="!tableform.$visible" ng-click="tableform.$show()">
-            <span class="btn-icon"><i class="fa fa-pencil"></i></span>Edit
-          </button>
-          <div class="btn-group pull-right" ng-show="tableform.$visible">
-            <button type="submit" ng-disabled="tableform.$waiting" class="btn btn-link">
-              <span class="btn-icon"><i class="fa fa-check"></i></span>Save
-            </button>
-            <button type="button" ng-disabled="tableform.$waiting" ng-click="tableform.$cancel()" class="btn btn-link">
-              <span class="btn-icon"><i class="fa fa-times"></i></span>Cancel
+          <div class="job-role__actions">
+            <button type="button" class="btn btn-link" ng-show="!tableform.$visible" ng-click="tableform.$show()">
+              <span class="btn-icon"><i class="fa fa-pencil"></i></span>Edit
             </button>
+            <div class="btn-group" ng-show="tableform.$visible">
+              <button type="submit" ng-disabled="tableform.$waiting" class="btn btn-link">
+                <span class="btn-icon"><i class="fa fa-check"></i></span>Save
+              </button>
+              <button type="button" ng-disabled="tableform.$waiting" ng-click="tableform.$cancel()" class="btn btn-link">
+                <span class="btn-icon"><i class="fa fa-times"></i></span>Cancel
+              </button>
+            </div>
           </div>
-          <table class="table table-striped">
-            <thead>
-              <tr>
-                <th style="width:40%">Funder</th>
-                <th style="width:15%">Type</th>
-                <th style="width:15%">% Amount</th>
-                <th style="width:15%">Absolute Amount</th>
-                <th style="width:15%" ng-show="tableform.$visible">Action</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr ng-repeat="(key, funder) in jobroles.editData['new_role_id']['funders']">
-                <td>
-                  <!--show funder in non-edit mode-->
-                  <span ng-show="!tableform.$visible">{{funder.funder_id.sort_name}}</span>
-                  <!--show ui-select for funder in edit mode-->
-                  <span ng-show="tableform.$visible">
-                    <ui-select class="funder-select"
-                      prevent-animations
-                      theme="civihr-ui-select"
-                      ng-model="funder.funder_id">
-                      <ui-select-match prevent-animations allow-clear placeholder="Select a funder" class="ui-select-match">
-                        {{$select.selected.sort_name}}
-                      </ui-select-match>
-                      <ui-select-choices
+          <div class="table-tab">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th style="width:40%">Funder</th>
```

Comment:
Just a comment: multiple sources state that the width attribute is not part of the HTML5 and using inline styles and classes are the recommended way to solve this issue. So this is perfectly fine.

[Source](http://help.simplytestable.com/errors/html-validation/the-x-attribute-on-the-y-element-is-obsolete-use-css-instead/the-width-attribute-on-the-td-element-is-obsolete-use-css-instead/)